packages/models-library/src/models_library/api_schemas_webserver/folders_v2.py

```diff
@@ -45,7 +45,7 @@ class CreateFolderBodyParams(InputSchema):
 
 class PutFolderBodyParams(InputSchema):
     name: IDStr
-    parent_folder_id: FolderID | None
+    parent_folder_id: FolderID | None = None
```

Comment:
this is a change in syntax. @matusdrobuliak66?

Reply:
Here, it's fine to have the default value as None. (We could also make it required since the frontend currently always fills this field.) However, we can go with the easier solution; both options are acceptable.